### ISSUE TITLE
📎 Cascading dying worker 

### ISSUE THREAD

**Issue by sebmck** [rome/tools#1029]
When a worker dies, we fatal error on the main process, this causes all the other workers to also be killed, which can result in error messages being printed for each. We should suppress these errors, and also preferably, have a better original error message for why the worker died.

**Comment by katungi** [comment]
Hey @sebmck can you explain more on the task?

**Comment by ematipico** [comment]
@sebmck I believe this is fixed?

**Comment by jamiebuilds** [comment]
Did this commit fix this issue? https://github.com/rome/tools/pull/1389/commits/78521b127a97b39c9fa3abf022ffa120c8250a3b

**Comment by jamiebuilds** [comment]
Answer:

> not necessarily, if there's an error in the binary transport layer then it can be hidden

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
